modal/app.py

```diff
@@ -240,6 +245,26 @@ async def deploy(self, name: str, namespace, object_entity: str) -> str:
         deploy_response = await retry_transient_errors(self._client.stub.AppDeploy, deploy_req)
         return deploy_response.url
 
+    async def spawn_container(
+        self, program: str, *args: str, shell: bool = False, image: Optional["_Image"] = None, mounts=[]
```

Comment:
minor preference for expecting an array of strings here rather than using `*args` here. makes it more consistent with `subproces`: https://docs.python.org/3/library/subprocess.html

Reply:
I was using the [asyncio.subprocess API](https://docs.python.org/3/library/asyncio-subprocess.html#creating-subprocesses) as reference. Surprising that the two are inconsistent!

Reply:
ok yeah maybe `*args` is more common in recent code. alright seems fine then!

Reply:
It's probably because `subprocess` has the same command for both `exec` and `shell` forms, whereas they're different in `asyncio.subprocess`. Probably makes sense for us to do that too—something like `spawn_sandbox` and `spawn_sandbox_shell` maybe?